frontend/components/product-list/sections/FilterableProductsSection/useFacets.tsx

```diff
@@ -43,5 +43,18 @@ export function useFilteredFacets(productList: ProductList) {
       return usefulFacets;
    }, [facets, infoNames, isItemTypeProductList]);
 
+   if (productList.type === ProductListType.AllTools) {
+      const excludedToolsFacets = [
+         'facet_tags.Item Type',
+         'facet_tags.Capacity',
+         'device',
+         'facet_tags.Device Brand',
+         'facet_tags.Device Category',
+         'facet_tags.Device Type',
+         'facet_tags.OS',
```

Comment:
I tried looking for that but  I am not sure.
@danielbeardsley @masonmcelvain @dhmacs and @sterlinghirsh do you know if we can do this?

Reply:
I agree that it seems better to just *not* include the facets in the indexed documents if we don't want them to be facets, but there may be more to it than that. Also, a single incorrect record out of thousands will end up showing these facets, so it may be better to be safe.

Reply:
I think we will come up with a more general way to do this in the future, perhaps through the strapi UI. I'm not too worried about it for now. We have a whitelist of facets at the top of this file. We currently have a handful of tools that have a worksin associated with them, which is why we're getting these device fields. We could definitely change the index to ignore worksin on tools. I do think we should drop Part or Kit too, though.